### ISSUE TITLE
Multiple colour contrast accessibility violations

### ISSUE THREAD

**Issue by jh409** [AJFaraday/buggy_library#5]
There are seven contrast errors reported by WebAIM's Wave accessibility plugin on Chrome 60.

These are present on the delete buttons:

<img width="293" alt="screen shot 2017-08-16 at 19 09 29" src="https://user-images.githubusercontent.com/5544479/29378318-846d54e2-82b6-11e7-9188-e9eaae2c26e9.png">
